### PR TITLE
Updated cartfile, added camera vc

### PR DESCRIPTION
**Pull request by steven-t-h**
Updated cartfile to remove --use-ssh flag, added Camera VC so show current camera features in SDK.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,13 +20,13 @@ Then to add the framework, create the file `Cartfile` in your app's repository w
...
 # Require the iOS framework of the Yuneec SDK
-github "YUNEEC/Yuneec-SDK-iOS" >= 0.3.1
+github "YUNEEC/Yuneec-SDK-iOS" >= 0.5.7
 ```
 
 Then, to pull in the library and build it, run Carthage in your app's repository:
 
 ```
-carthage update --use-ssh
+carthage update
```
Comment (julianoes) [inline README.md]: Good point.

File: SdkExample/Controllers/ActionsViewController.h [hunk 2]
```diff
@@ -2,8 +2,8 @@
 //  ActionsViewController.h
 //  SdkExample
 //
-//  Created by Julian Oes on 31/08/16.
-//  Copyright © 2016 Julian Oes. All rights reserved.
+//  Created by Steven Hall on 9/13/17.
+//  Copyright © 2017 yuneec. All rights reserved.
```
Comment (julianoes) [inline SdkExample/Controllers/ActionsViewController.h]: 😞

Comment (julianoes) [reply SdkExample/Controllers/ActionsViewController.h]: Na no worries 😄

File: Cartfile [hunk 3]
```diff
@@ -1,2 +1,2 @@
 # Require the Objective-C wrappers of the Yuneec SDK
-github "YUNEEC/Yuneec-SDK-iOS" >= 0.2.0
+github "YUNEEC/Yuneec-SDK-iOS" >= 0.5.7
```
Comment (julianoes) [inline Cartfile]: You don't have 0.6.1 yet, right?

File: SdkExample/Controllers/ActionsViewController.h (hunk 2 above)
Comment (steven-t-h) [reply SdkExample/Controllers/ActionsViewController.h]: I let Xcode clean up some of the code, we can certainly put your name back in there if you'd like haha

File: Cartfile (hunk 3 above)
Comment (steven-t-h) [reply Cartfile]: No, currently only 0.5.7 for iOS SDK.

**Pull request merged**